.github/workflows/publish-ollama.yaml

```diff
@@ -0,0 +1,45 @@
+name: Ollama
+on:
+  push:
+    tags:
+      - 'v*.*.*'
+jobs:
+  publish:
+    name: Build and push ollama container
+    runs-on: ubuntu-latest
+    permissions:
+      contents: 'read'
+      id-token: 'write'
+      packages: 'write'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          # list of Docker images to use as base name for tags
+          images: |
+            ghcr.io/pluralsh/ollama
+          # generate Docker tags based on the following events/attributes
+          tags: |
+            type=semver,pattern={{version}}
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Login to GHCR
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Build and push
```

Comment:
I think this action should ultimately be a matrix gh action with the model variants we want to support.  We can start small w/ just llama3.1 8b i think and then add additional images for other ones.